double-dispatch/src/main/java/com/iluwatar/doubledispatch/SpaceStationMir.java

```diff
@@ -45,29 +47,29 @@ public void collision(GameObject gameObject) {
 
   @Override
   public void collisionResolve(FlamingAsteroid asteroid) {
-    LOGGER.info("{} hits {}. {} is damaged! {} is set on fire!", asteroid.getClass().getSimpleName(),
+    LOGGER.info(AppConstants.HITS," {} is damaged! {} is set on fire!", asteroid.getClass().getSimpleName(),
         this.getClass().getSimpleName(), this.getClass().getSimpleName(), this.getClass().getSimpleName());
     setDamaged(true);
     setOnFire(true);
   }
 
   @Override
   public void collisionResolve(Meteoroid meteoroid) {
-    LOGGER.info("{} hits {}. {} is damaged!", meteoroid.getClass().getSimpleName(),
+    LOGGER.info(AppConstants.HITS," {} is damaged!", meteoroid.getClass().getSimpleName(),
         this.getClass().getSimpleName(), this.getClass().getSimpleName());
     setDamaged(true);
   }
 
   @Override
   public void collisionResolve(SpaceStationMir mir) {
-    LOGGER.info("{} hits {}. {} is damaged!", mir.getClass().getSimpleName(),
+    LOGGER.info(AppConstants.HITS," {} is damaged!", mir.getClass().getSimpleName(),
         this.getClass().getSimpleName(), this.getClass().getSimpleName());
     setDamaged(true);
   }
 
   @Override
   public void collisionResolve(SpaceStationIss iss) {
-    LOGGER.info("{} hits {}. {} is damaged!", iss.getClass().getSimpleName(),
```

Comment:
I noticed this change resulted into more arguments than the placeholder. This resulting log statement as below
```
04:54:48.252 [main] INFO com.iluwatar.doubledispatch.SpaceStationMir -  {} is damaged! {} is set on fire! hits FlamingAsteroid.
04:54:48.278 [main] INFO com.iluwatar.doubledispatch.SpaceStationMir -  {} is damaged! {} is set on fire! hits FlamingAsteroid.

```

Reply:
Fixed in commit https://github.com/iluwatar/java-design-patterns/commit/195a735814b0b59a43998d9b0bba051d4b43ebcb